server/src/main/java/org/opensearch/search/aggregations/bucket/filterrewrite/PointTreeTraversal.java

```diff
@@ -0,0 +1,223 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.search.aggregations.bucket.filterrewrite;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.lucene.index.PointValues;
+import org.apache.lucene.search.CollectionTerminatedException;
+import org.apache.lucene.search.DocIdSetIterator;
+import org.opensearch.common.CheckedRunnable;
+
+import java.io.IOException;
+import java.util.function.BiConsumer;
+
+import static org.apache.lucene.search.DocIdSetIterator.NO_MORE_DOCS;
+
+/**
+ * Utility class for traversing a {@link PointValues.PointTree} and collecting document counts for the ranges.
+ *
+ * <p>The main entry point is the {@link #multiRangesTraverse(PointValues.PointTree, Ranges,
+ * BiConsumer, int)} method
+ *
+ * <p>The class uses a {@link RangeCollectorForPointTree} to keep track of the active ranges and
+ * determine which parts of the tree to visit. The {@link
+ * PointValues.IntersectVisitor} implementation is responsible for the actual visitation and
+ * document count collection.
+ */
+final class PointTreeTraversal {
+    private PointTreeTraversal() {}
+
+    private static final Logger logger = LogManager.getLogger(Helper.loggerName);
```

Comment:
Sorry didn't see this before - why are we setting all the loggers to Helper.class ?

Reply:
I want the logs emitted from this package to have the same name.